getDataFRF.py

```diff
@@ -2426,7 +2437,7 @@ def getStwaveField(self, var, prefix, local=True, ijLoc=None):
         assert field[var].shape[0] == len(field['time']), " the indexing is wrong for pulling down bathy"
         return field
 
-    def getWaveSpecSTWAVE(self, prefix, gaugenumber, local=True):
+    def getWaveSpecSTWAVE(self, prefix, gaugenumber, local=True, model='STWAVE'):
```

Comment:
Try to do something like this.  I think its a good way to not break things, but will move things forward as well! 

    def getWaveSpecSTWAVE(self, prefix, gaugenumber, local=True, model='STWAVE'):
        warnings.warn('Using depricated function name')
        return self.getWaveSpecSTWAVE(prefix, gaugenumber, local, model)

    def getWaveSpecModel(self, prefix, gaugenumber, local=True, model='STWAVE'):

Reply:
also this would work for the STWAVE field function

Reply:
Should be like this instead right?

return self.getWaveSpecModel(prefix, gaugenumber, local, model)

Reply:
Made the change.